lib/Settings/Settings.cpp

```diff
@@ -29,10 +29,16 @@
 
 #include "Settings.h"
 
+#include <ArduinoJson.h>
+
 static inline Settings::SettingTypeSet setFor(const SettingType type) {
   return Settings::SettingTypeSet().set(type);
 }
 
+static inline String maskSensible(const String &val, const bool sensible) {
+  return (sensible ? val : F("xxx"));
```

Comment:
I would prefer `"***"`.

Reply:
I'm actually not sure if we should leave these values out at all. The deserialization is completely fine with missing values.

Reply:
But we need to find a way to tell the user on the web interfaces if a password is already configured.

Reply:
I would opt for "there is always a password. Either the default or the user-choosen one". So the user can only change or reset it.